tensorboard/webapp/feature_flag/views/feature_flag_page_container.ts

```diff
@@ -14,15 +14,89 @@ limitations under the License.
 ==============================================================================*/
 import {Component} from '@angular/core';
 import {Store} from '@ngrx/store';
+import {Observable} from 'rxjs';
+import {map, withLatestFrom} from 'rxjs/operators';
 import {State} from '../../app_state';
-import {getFeatureFlags} from '../store/feature_flag_selectors';
+import {
+  allFeatureFlagOverridesReset,
+  featureFlagOverrideChanged,
+  featureFlagOverridesReset,
+} from '../actions/feature_flag_actions';
+import {
+  getDefaultFeatureFlags,
+  getOverriddenFeatureFlags,
+} from '../store/feature_flag_selectors';
+import {FeatureFlags} from '../types';
+import {
+  FeatureFlagState,
+  FeatureFlagStatus,
+  FeatureFlagStatusEvent,
+} from './types';
+
 @Component({
   selector: 'feature-flag-page',
   template: `<feature-flag-page-component
-    [featureFlags]="featureFlags$ | async"
+    [featureFlagStatuses]="featureFlags$ | async"
+    (flagChanged)="onFlagChanged($event)"
+    (allFlagsReset)="onAllFlagsReset()"
   ></feature-flag-page-component>`,
 })
 export class FeatureFlagPageContainer {
   constructor(private readonly store: Store<State>) {}
-  readonly featureFlags$ = this.store.select(getFeatureFlags);
+
+  readonly featureFlags$: Observable<FeatureFlagState<keyof FeatureFlags>[]> =
+    this.store.select(getDefaultFeatureFlags).pipe(
+      withLatestFrom(this.store.select(getOverriddenFeatureFlags)),
+      map(([defaultFeatureFlags, overriddenFeatureFlags]) => {
+        return Object.entries(defaultFeatureFlags).map(
```

Comment:
To consider for a followup PR (in order to keep this one from growing any larger): Can we disable the UI for any non-boolean feature flags until we support them? Like, render a line for those properties but don't allow the user to interact with the line and don't show "Enabled"/"Disabled" but some other text like "Not Supported in this UI" or something?

Or, if that's too difficult, just don't render them at all on the page.

Either way, I'd prefer not to be able to "Enable/Disable" these non-boolean feature flags in the UI.

Reply:
I've gone ahead and disabled these for now

![Screen Shot 2022-08-22 at 1 14 17 PM](https://user-images.githubusercontent.com/78179109/186010593-667c5b94-d9f6-4719-859e-5d89a91a795e.png)
.